include/kos/gmon.h

```diff
@@ -0,0 +1,63 @@
+/* KallistiOS ##version##
+
+   kos/gmon.h
+   Copyright (C) 2024 Andy Barajas
+
+*/
+
+/** \file    kos/gmon.h
+    \brief   GPROF support.
+    \ingroup debugging_gprof
+
+    This file provides utilities for profiling applications using gprof.
+
+    \author Andy Barajas
+*/
+
+#ifndef __KOS_GMON_H
+#define __KOS_GMON_H
+
+#include <sys/cdefs.h>
+__BEGIN_DECLS
+
+#include <stdbool.h>
+
+/** \defgroup debugging_gprof GPROF
+    \brief    Utilities for GPROF
+    \ingroup  debugging
+
```

Comment:
Could you write a brief little summary of how to enable this and do the profiling here if it's not too long?